Rakefile

```diff
@@ -1,3 +1,27 @@
 require "rspec/core/rake_task"
 RSpec::Core::RakeTask.new(:spec)
+
+desc "Build H3 C library"
+task :build do
+  unless File.exists?("ext/h3/src/Makefile")
+    `git submodule update --init --recursive`
+    print "Building h3..."
+    `cd ext/h3; make > /dev/null 2>&1`
+    puts " done."
+  end
+end
+
+desc "Remove compiled H3 library"
+task :clean do
+  File.delete("ext/h3/src/Makefile") if File.exists?("ext/h3/src/Makefile")
```

Comment:
I wonder... the `Makefile` of H3 has a `clean` target. It does not delete the directories themselves, but wouldn't it be enough to delegate to their `make clean` and assume they know what it needs to be cleaned?

Reply:
Good thinking - I'll see if it works 👍

Reply:
Tried it - it doesn't seem to remove any of the compiled code...